constraintlayout/core/src/main/java/androidx/constraintlayout/core/motion/Motion.java

```diff
@@ -1727,15 +1735,20 @@ public boolean setValue(int id, int value) {
 
     @Override
     public boolean setValue(int id, float value) {
+        if (MotionType.TYPE_QUANTIZE_MOTION_PHASE == id) {
+            mQuantizeMotionPhase =  value;
+        } else if (MotionType.TYPE_STAGGER == id) {
+            mMotionStagger = value;
+        }
```

Comment:
shouldn't we return true in these cases?

Reply:
It does not matter but to be consistent change